src/igvfd/tests/test_audit_measurement_set.py

```diff
@@ -421,6 +421,48 @@ def test_audit_missing_institutional_certification(
     )
 
 
+def test_audit_inconsistent_controlled_access(
+    testapp,
+    measurement_set,
+    sequence_file,
+    controlled_sequence_file_2,
+    tissue,
+    institutional_certificate
+):
+    # No audit when the file and the sample's IC
+    # are both uncontrolled (or both controlled)
+    testapp.patch_json(
+        sequence_file['@id'],
+        {
+            'file_set': measurement_set['@id']
+        }
+    )
+    testapp.patch_json(
+        institutional_certificate['@id'],
+        {
+            'samples': [tissue['@id']]
+        }
+    )
+    res = testapp.get(measurement_set['@id'] + '@@audit')
+    assert all(
+        error['category'] != 'inconsistent controlled access'
+        for error in res.json['audit'].get('ERROR', [])
+    )
+    # Audit if any file's controlled access is different
+    # from the sample's IC, or the files are mixed access.
+    testapp.patch_json(
+        controlled_sequence_file_2['@id'],
+        {
+            'file_set': measurement_set['@id']
+        }
+    )
+    res = testapp.get(measurement_set['@id'] + '@@audit')
+    assert any(
+        error['category'] == 'inconsistent controlled access'
+        for error in res.json['audit'].get('ERROR', [])
+    )
+
+
```

Comment:
can we also add a case where files have the same access status but it's different from IC?